src/openforms/authentication/utils.py

```diff
@@ -26,3 +29,17 @@ def store_auth_details(submission: Submission, form_auth: FormAuth) -> None:
     ), f"Unexpected auth attribute {attribute} specified"
 
     AuthInfo.objects.update_or_create(submission=submission, defaults=form_auth)
+
+
+def store_registrator_details(
+    submission: Submission, registrator_auth: BaseAuth
```

Comment:
I'd like a more specific type hint than `BaseAuth` which also contains the `skip_subject_info` key with boolean value

Reply:
If there is a `skip_subject_info` in the dictionary in the session then it won't reach here, and a new dict conforming `BaseAuth` is created in `set_auth_attribute_on_session()`. I could make a type of what is in the session but it would be like a `Union[BaseAuth, TypedDictWithOnlySkipSubjectInfo]`

Reply:
```
class SkipSubjectInfo(TypedDict):
    skip_subject_info: bool

RegistratorInfo = Union[BaseAuth, SkipSubjectInfo]
```
is probably the compromise to go for then - you can then use the `RegistratorInfo` type hint and editors will pick it up correctly.

Reply:
But `store_registrator_details()` doesn't do anything with `skip_subject_info`, so that union would be invalid in that function signature.